Task 3/Insertion Sort/insertion sort_c

```diff
@@ -0,0 +1,43 @@
+
```

Comment:
Please add a multiline comment here mentioning sample input & output cases alongwith a short description of what insertion sort is.

Reply:
okay
 corrected. Thank you for pointing it out.